README.md

```diff
@@ -172,6 +178,131 @@ The OSIsoft connector now has the ability to support OMF. To enable OMF follow t
    1. IMPORTANT: The PI Data Archive Server can disconnect for various reasons. Make sure this is connected if you have a connection issue.
    1. Click confirm and finish running the utility.
 
+## OSIsoft Cloud Service
+
+### OCS Account Setup
+The OSIsoft connector supports an OCS connection. To make use of this feature follow the below steps.
+1. First create an OCS account.
+1. Under the namespace tab, create an OCS namespace of your choosing.
+   1. Save the namespace for later use. It will be needed in another step.
+1. Under the client tab, create a client.
+   1. Select "Client-Credentials" as the client type.
+   1. Click the "Add Client button.
+   1. Give the client a name. NOTE: the client will be the flexy device.
+   1. Select the "Account Administrator" role which will check all the role selection boxes.
+   1. Make sure the token lifetime is set to 3600 seconds.
+   1. Add a secret decription of your choosing.
+   1. Check the box that says "Never Expires"
+   1. Click "Add"
+   1. Copy both the client ID and client secret to use later.
+1. Under the connections tab, create a new client connection
+   1. Select OMF as the connection type.
+   1. Click the "Add Connection" button.
+   1. Enter the connection name, and hit next.
+   1. Select the client you just created and hit next.
+   1. Select the namespace you created and hit next.
+   1. Click the "Save" button.
+   1. Expand the dropdown menu that now shows up under the list of clients.
+   1. Click on your new client, more details will be available on the right hand panel.
+1. Obtain the Client ID.
+   1. Navigate to the namespace tab
+   1. Select your namespace
+   1. Click the "Display Details" button.
+   1. Copy the Account id for later use. This is your tenant ID.
```

Comment:
should be properly numbered

Reply:
it changes 1 to whatever the correct number is